lessons/Algebra2/exploring-linear-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,265 @@
+
+[.lesson-instruction]
+- Working in pairs or small groups, complete questions 3 and 4 (making displays and suggesting possible relationships) in @printable-exercise{exploring-states.adoc}.
+- **Do not go on to question 5!** We'll be returning to that later in the lesson.
+
+Note: During this activity, don't let students just make scatter plot for random pairs of columns. Instead, encourage them to first _think about which columns might be related_, and then create the scatter plot to search for this relationship. The dataset is designed so that students will quickly begin searching for relationships between varying levels of education and income, and there are linear relationships in each of these.
+
+=== Synthesize
+
+- Share your scatter-plots with one another (copying and pasting scatter plot displays into a shared document, for example, and then labeling those displays). What possible relationships did you find in question 3?
+- Did you and your classmates commonly use any words to describe the relationships you observed?
+** Teacher note: the next section of the lesson is all about identifying form, direction, and strength. Students will acquire the formal vocabulary that data scientists use to assess relationships.
+
+== Describing Patterns @duration{15 minutes}
+
+=== Overview
+Students identify and make use of correlations in scatter plots, learning to characterize their form as being linear, curved, or showing no clear pattern. They learn how to describe the @vocab{strength} of correlations, and that linear patterns have direction.
+
+=== Launch
+
+Scatter plots let us visualize the relationship between two columns. If no relationship exists, the points in the scatter plot just appear as a shapeless cloud. But if there _is_ a relationship, the points will form some kind of pattern. When we build scatter plots, we are searching for patterns - or @vocab{correlations} between two quantitative variables.
+
+These patterns can be described using three qualities: form, direction, and strength.
+
+@vocab{Form} refers to the shape of the pattern. Sometimes a pattern looks like a straight line, or a curve, or a hockey stick. And sometime, there's no clear shape at all!
+
+[.lesson-point]
+Form indicates whether a relationship is linear, non-linear or undefined.
+
+[cols="^1a,^1a,^1a", stripes="none"]
+|===
+| @image{images/1b1.gif, 250}
+| @image{images/2NL.gif, 250}
+| @image{images/B.gif, 250}
+
+| Some patterns are @vocab{linear}, and cluster around a straight line sloping up or down.
+| Some patterns are **non-linear**, and may look like a curve or an arc.
+| And sometimes there is **no relationship** or pattern at all!
+|===
+
+[.lesson-point]
+Linear relationships have direction
+
+For straight-line relationships, we can talk about _direction._
```

Comment:
[This is a term of art](https://www.sciencedirect.com/topics/mathematics/line-relationship), but I'm not opposed to other wording. Can you suggest something?

Reply:
If the relationship clusters around a straight line, we can talk about _direction_.

I think straight-line relationship doesn't sit well with me here because it's a single sentence following a lesson point which says almost the same thing at a time when we're introducing linear relationship as a a concept and it doesn't actually mean that we're looking for a straight line.

Reply:
Love it. Done.